### DISCUSSION TITLE
[Fixed] Inputting .3 into NumberInput turns into 3

### DISCUSSION THREAD

**Discussion by hunterwilhelm** [mantinedev/mantine#1519] [Feedback]
Hello, my users are complaining that they cannot put `.3` into the NumberInput - it only shows `3`. However, it does work by inputting `0.3`. Is this expected?

I tried it out on this part of the website. Inputting the `.` first is ignored.
> https://mantine.dev/core/number-input/#decimal-steps

Thanks

tags: decimal, point, dot

**Comment by rtivital** [comment]
Thanks for reporting, I'll see if it is possible to add leading `.` support

**Reply by hunterwilhelm** [reply]
It seems to be working in the latest version, thank you!
